game/settings.py

```diff
@@ -25,6 +25,9 @@ class Settings:
 
     default_start_type: str = "Cold"
 
+    # Mission specific
+    mission_length: int = 90
```

Comment:
Nit on the name: how about `desired_player_mission_duration`?

Make it a `timedelta` as well (from the `datetime` package). Takes the guesswork out of the units :)